### PR TITLE
🐛 Add back `service-account-credentials`

### PR DESCRIPTION
**Pull request by czunker**
Signed-off-by: Christian Zunker <christian@mondoo.com>

### CODE REVIEW

File: docker-image/action.yaml [hunk 1]
```diff
@@ -36,3 +39,5 @@ runs:
     - ${{ inputs.score-threshold }}
     - --log-level
     - ${{ inputs.log-level }}
+  env:
+    MONDOO_CONFIG_BASE64: ${{ inputs.service-account-credentials }}
```
Comment (imilchev) [inline docker-image/action.yaml:43]: So when `service-account-credentials` isn't set, then the environment from the caller of the action is used? I guess that's why this works?

Comment (czunker) [reply docker-image/action.yaml:43]: Yes, that's how I also think it works. I couldn't find explicit docs about it. But having a look at how the two different steps are called, the assumption seems valid:

The first step w/o env var, but w/ arg:
image

The second step w/ env, but w/o arg:
image

And both seem to report to the same space.

**Review by imilchev — APPROVED** [review]

File: terraform-plan/action.yaml [hunk 2]
```diff
@@ -36,3 +39,5 @@ runs:
     - ${{ inputs.score-threshold }}
     - --log-level
     - ${{ inputs.log-level }}
+  env:
+    MONDOO_CONFIG_BASE64: ${{ inputs.service-account-credentials }}
```
Comment (chris-rock) [inline terraform-plan/action.yaml:43]: Isn't that overriding the env variable with an empty value when the user sets:

```
env:
          MONDOO_CONFIG_BASE64: ${{ secrets.MONDOO_SERVICE_ACCOUNT }}
```

Comment (czunker) [reply terraform-plan/action.yaml:43]: No, it looks like the env var set when calling the action gets precedence over the env var I pre-set here.

I included the env var case and the args case in this test: https://github.com/mondoohq/actions/blob/5bb92b8e6b440403349d39f074e387b780b194b8/.github/workflows/docker-image.yaml

The tests look good:
https://github.com/mondoohq/actions/actions/runs/3713922613/jobs/6297163046

**Pull request merged**